stable/pomerium/templates/ingress.yaml

```diff
@@ -16,29 +16,49 @@ spec:
   tls:
     - secretName: {{ default .Values.ingress.secretName .Values.ingress.secret.name}}
       hosts:
-        - {{ printf "*.%s" .Values.config.rootDomain | quote }}
+        - {{ printf "authorize.%s" .Values.config.rootDomain | quote }}
+        - {{ printf "authenticate.%s" .Values.config.rootDomain | quote }}
+      {{- range .Values.config.policy }}
```

Comment:
Can you make this mutually exclusive with the explicit ingress.hosts list?

Reply:
Done

Reply:
👍